app/editor/src/features/content/form/interfaces/IContentForm.tsx

```diff
@@ -10,8 +10,8 @@ import {
   IContentTonePoolModel,
   IFileReferenceModel,
   ITimeTrackingModel,
-  IWorkOrderModel,
 } from 'hooks/api-editor';
+import { IWorkOrderModel } from 'store/slices/workorder/interfaces';
```

Comment:
I don't understand the need for a different interface.

Reply:
A lot of the changes were done as the interfaces were stored in /admin and had been relocated. Did a mass update imports - will look into these